packages/hydrogen/package.json

```diff
@@ -87,6 +87,7 @@
   "peerDependencies": {
     "@remix-run/react": "^2.1.0",
     "@remix-run/server-runtime": "^2.1.0",
+    "@remix-run/dev": "^2.1.0",
```

Comment:
cc @frandiox I had to add this in for typecheck to pass. Make sense?

Reply:
Hmm can you try to move it to `devDependencies` and see if that works? Otherwise I guess it's not the end of the world, but we only use this in Hydrogen to import the Remix' `Preset` type.